### ISSUE TITLE
Two-page poster PDF

### ISSUE THREAD

**Issue by drakulis** [gregsramblings/google-cloud-4-words#23]
I really love this cheat sheet but when I print it on standard A4 page the texts are really small and you have to strain your eyes to read them.. 
Currently I'm splitting PNG file in image editor into two and print them separately but it would be nice to have two-page poster version of PDF.

**Comment by gregsramblings** [comment]
Yeah, I completely agree.  The list has become too big and now you need actual poster paper!  Did you try the Brochure.pdf? It's designed to print on both sides and fold (tri-fold).

**Comment by lukwam** [comment]
It's a real problem, all these products!

**Comment by drakulis** [comment]
@gregsramblings yes, I saw the brochure, but as I've wrote, it is organized different way and it is intended to be folded. It is cool as a ad or flyer but I don't want to fold it and unfolded looks a bit unnatural - beginning is at the end of the page. 

If it doesn't cost you lots of time, it would be great to have two-page version of the poster - with more natural order of the data

**Comment by gregsramblings** [comment]
Yeah, I think we need to do that. I'm going to have our designer experiment a bit with some different formats. A two-page poster would be good. How would you like to split it? Maybe page 2 is Firebase/Maps/G Suite/Resources and I try to fit all core GCP on page 1?

**Comment by drakulis** [comment]
Sounds good 👍
